icon

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
```

Comment:
Was this file accidentally committed?

Reply:
No, but it's possible Git hasn't committed what I thought it did. This is so that the app icon is available to copy and paste into the app. I'll check